miso-web/src/main/webapp/scripts/box_visualization.js

```diff
@@ -20,686 +20,696 @@
  *
  */
 
-/*
- * A BoxPosition represents a single position (cell) in the Box visualization (table).
- * It is responsible for handling the functionality of a cell. This includes changing the title and image
- * associated with the position.
- */
-var BoxPosition = function(opts) {
-  var self = {};
+(function(Box, $, undefined) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Do not use "undefined" to declare a parameter - use another name. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2137)